Formula/postgresql@10.rb

```diff
@@ -59,7 +55,17 @@ def install
     # to inside the SDK, so we need to use `-iwithsysroot` instead
     # of `-I` to point to the correct location.
     # https://www.postgresql.org/message-id/153558865647.1483.573481613491501077%40wrigleys.postgresql.org
-    ENV.prepend "LDFLAGS", "-R#{lib}/postgresql"
+    if DevelopmentTools.clang_build_version >= 1000
+      inreplace "configure",
+                "-I$perl_archlibexp/CORE",
+                "-iwithsysroot $perl_archlibexp/CORE"
+      inreplace "contrib/hstore_plperl/Makefile",
+                "$(perl_archlibexp)/CORE",
+                "-iwithsysroot $(perl_archlibexp)/CORE"
+      inreplace "src/pl/plperl/GNUmakefile",
+                "$(perl_archlibexp)/CORE",
+                "-iwithsysroot $(perl_archlibexp)/CORE"
+    end
```

Comment:
Has this regressed? It was fixed in 10.6. In fact, it seems like it was backported to 9.5 and 9.6.

On that note, I also suspect the `tclconfig` stuff is also no longer needed under any version after https://github.com/postgres/postgres/commit/5e22171310f8d7c82219a6b978440e5144e88683, but I haven't tested it.

Reply:
The build for `postgresql@10` was broken on something about `hstore_plperl` until I copied the `inreplace`s from the 9.5/9.6 formula into here.

Reply:
Hmm, ok. Bit unusual since `hstore_plperl` hasn't been touched on the 10.x branch since 2018.

Reply:
> On that note, I also suspect the tclconfig stuff is also no longer needed under any version after postgres/postgres@5e22171, but I haven't tested it.

I'll test this locally first and if it works, I'll open a separate PR for it.

Reply:
Yeah, I'll merge this as-is and we can experiment in a separate PR. I'm just nitpicking on cleaning up workarounds that may no longer be needed - they pile up a bit so it's nice when they can be removed (like with the LLVM formula).